src/odoo_sort_manifest_depends/sort_manifest_deps.py

```diff
@@ -7,11 +7,18 @@
 
 import click
 from click import command, option
+from diskcache import Cache
 from manifestoo_core.addon import Addon, is_addon_dir
 from manifestoo_core.core_addons import is_core_ce_addon, is_core_ee_addon
+from manifestoo_core.metadata import addon_name_to_distribution_name
 from manifestoo_core.odoo_series import OdooSeries
+from requests import head
 
 NAME_DEFAULT_CATEGORY = "Default"
+OCA_ADDON_URL = "https://wheelhouse.odoo-community.org/oca-simple/"
+REQUEST_TIMEOUT = 2  # s
+
+other_addons_category_cache = Cache("~/.cache/odoo-sort-manifest-depends")
```

Comment:
Use [platformdirs.user_cache_dir](https://pypi.org/project/platformdirs/) for portability.